submarine-dist/src/assembly/distribution.xml

```diff
@@ -138,10 +138,6 @@
         <include>submarine-submitter-k8s-${project.version}.jar</include>
       </includes>
     </fileSet>
-    <fileSet>
-      <directory>../submarine-server/server-submitter/submitter-k8s/target/dependency</directory>
-      <outputDirectory>/lib/submitter/k8s</outputDirectory>
-    </fileSet>
```

Comment:
If we don't copy the submitter jars, can it still submit jobs?

Reply:
yes, currently has one jar
```
➜  lib git:(SUBMARINE-672) ✗ pwd
/home/kobe/git/submarine/submarine-dist/target/submarine-dist-0.5.0-hadoop-2.9/submarine-dist-0.5.0-hadoop-2.9/lib
➜  lib git:(SUBMARINE-672) ✗ ls submitter/k8s
submarine-submitter-k8s-0.5.0.jar
```
we copy  jars from `submarine-server/server-submitter/submitter-k8s/target/dependency` to `submarine-dist-0.5.0-hadoop-2.9/lib/submitter/k8s`, but those jars is same with `submarine-dist-0.5.0-hadoop-2.9/lib`